docs/views/privacy-policy.html

```diff
@@ -29,14 +29,12 @@ <h1 class="govuk-heading-xl">Privacy policy</h1>
       <ul class="govuk-list govuk-list--bullet">
         <li>questions, queries or feedback you leave</li>
         <li>your email address, if you contact us</li>
-        <li>your Internet Protocol (IP) address, and which version of web browser you used</li>
-        <li>information on how you use the site, using <a href="/docs/cookies">cookies</a> and page-tagging techniques</li>
+        <li>your Internet Protocol (IP) address, which is anonymised before it's stored in our server logs</li>
```

Comment:
This may be true in the future if we migrate from Heroku to Paas and start persisting server logs, but right now since we've remove Analytics this line is no longer relevant.

Reply:
Thanks for letting me know - fixed in https://github.com/alphagov/govuk-prototype-kit/pull/845/commits/3b58e84a4183ba3ff3f93bf721fda777b4ffd18b